tests/test_ampOffset.py

```diff
@@ -372,6 +373,7 @@ def testAmpOffset(self, valueType):
             config.doBackground = False
             config.doDetection = False
             config.ampEdgeWidth = 12  # Given 100x51 amps in our mock detector.
+            config.doApplyAmpOffset = True  # Updates the exposure in place.
```

Comment:
Can this have a trivial test that with `doApplyAmpOffset=False` the image after being measured is the same as it was before?

Reply:
Done!